phaseshifts/wrappers.py

```
@@ -0,0 +1,733 @@
+                        "\nRemoving pi/2 jumps in '%s':\n" % os.path.basename(filename)
+                    )
+                    phsh = Conphas(
+                        input_files=[phasout_files[i]],
+                        output_file=filename,
+                        formatting=out_format,
+                        lmax=lmax_dict[phaseshift],
+                    )
+                    phsh.calculate()
+
+        except AttributeError:
+            raise AttributeError("MTZ_model has no NFORM (0-2) specified!")
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)